ts/IterableInboxMessageList.tsx

```diff
@@ -52,18 +55,56 @@ const IterableInboxMessageList = ({
       )
    }
 
+   function getInAppMessageFromViewToken(viewToken: ViewToken): IterableInAppMessage {
```

Comment:
What is viewToken?

Reply:
It's related to the RN FlatList component, it contains info about specific cells when tracking. [API docs](https://reactnative.dev/docs/0.65/viewtoken)